app/models/merchant.rb

```diff
@@ -12,4 +13,23 @@ def self.most_items_sold(quantity)
       group("merchants.id").
       first(quantity)
   end
+
+  def revenue
+    { "revenue" => calculate_revenue }
```

Comment:
I have a feeling they would prefer to have the query return in this format, rather than a separate formatting method, but I like that it is working, and everything looks very clean overall.